markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 15 Line 3_
```markdown
* * *

Added some
```
Added some review comment 2

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._

Reply:
response to comment 2

Reply:
2